### PR TITLE
Add samples/tutorials links to IQ# install guide

### PR DESCRIPTION
**Pull request by rmshaffer**
This PR adds a couple of links to the end of the IQ# installation guide, since the default QRNG quickstart doesn't really show anything Jupyter-specific.

Also a few other tweaks for consistency.

Edit: @geduardo recently merged similar changes, so this ends up being just tweaks to what he already did.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/1925002a028efaecf401ccac2aa222c3f025f6cc/articles/install-guide/pyinstall.md) | 94 | 76 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9beabc1a-03a4-4bf4-a6f0-f09728097379_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/qjupyter.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/1925002a028efaecf401ccac2aa222c3f025f6cc/articles/install-guide/qjupyter.md) | 87 | 71 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/876fa06a-61f7-4a2a-9aa1-1316279eb419_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/install-guide/qjupyter.md [hunk 1]
```diff
@@ -52,17 +52,23 @@ IQ# (pronounced i-q-sharp) is an extension primarily used by Jupyter and Python
...
 
 ## Next steps
 
-Now that you have installed the Quantum Development Kit in your preferred environment, you can write and run [your first quantum program](xref:microsoft.quantum.quickstarts.qrng).
+Now that you have installed the QDK for Q# Jupyter Notebooks, you can write and run [your first quantum program](xref:microsoft.quantum.quickstarts.qrng) by writing your Q# code directly within the Jupyter Notebook environment.
+
+## Samples
+
+For more examples of what you can do with Q# Jupyter Notebooks, please see:
+- [Intro to Q# and Jupyter Notebook](https://docs.microsoft.com/en-us/samples/microsoft/quantum/intro-to-qsharp-jupyter/)
```
Comment (cgranade) [inline articles/install-guide/qjupyter.md:73]:
```suggestion
- [Intro to Q# and Jupyter Notebook](https://docs.microsoft.com/samples/microsoft/quantum/intro-to-qsharp-jupyter/)
```

**Comment by rmshaffer** [comment]
@geduardo merged similar changes literally 10 minutes before I published this PR! I guess we are on the same wavelength. :-)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/1c6eafac107af14980a83e1c4a84bbea17acc7f7/articles/install-guide/pyinstall.md) | 94 | 76 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1b6555f5-f896-46de-9fb2-12f1f983bab6_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/qjupyter.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/1c6eafac107af14980a83e1c4a84bbea17acc7f7/articles/install-guide/qjupyter.md) | 87 | 71 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bceb90ef-2b90-4545-8bf4-77fceb9e0019_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/ccebbf71f97f85f0c8a864db0269640b7b5bcf36/articles/install-guide/pyinstall.md) | 94 | 76 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3d66e034-dbba-45ee-b683-cac2e200fb41_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/qjupyter.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/ccebbf71f97f85f0c8a864db0269640b7b5bcf36/articles/install-guide/qjupyter.md) | 87 | 73 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/eb74879a-e81f-4d78-8505-52639c698d97_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/64f007c09196e39ae210d6da5f0a720e27672158/articles/install-guide/pyinstall.md) | 94 | 76 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4d1329d3-d892-4b83-b5ab-7ad9fb0916b2_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/qjupyter.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/64f007c09196e39ae210d6da5f0a720e27672158/articles/install-guide/qjupyter.md) | 87 | 73 | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/35d09614-25f9-4f82-8fa0-bb4160ffb793_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Pull request merged**